src/Compilers/Core/Portable/Generated/Operations.xml.Generated.cs

```diff
@@ -3051,6 +3051,62 @@ public LazyObjectCreationExpression(IMethodSymbol constructor, Lazy<ImmutableArr
     }
 
     /// <summary>
+    /// Represents a C# or VB new/New anonymous object creation expression.
+    /// </summary>
+    internal abstract partial class BaseAnonymousObjectCreationExpression : Operation, IAnonymousObjectCreationExpression
+    {
+        protected BaseAnonymousObjectCreationExpression(bool isInvalid, SyntaxNode syntax, ITypeSymbol type, Optional<object> constantValue) :
+                    base(OperationKind.AnonymousObjectCreationExpression, isInvalid, syntax, type, constantValue)
+        {
+        }
+        /// <summary>
+        /// Explicitly-specified member initializers.
+        /// </summary>
+        public abstract ImmutableArray<IOperation> Initializers { get; }
+        public override void Accept(OperationVisitor visitor)
+        {
+            visitor.VisitAnonymousObjectCreationExpression(this);
+        }
+        public override TResult Accept<TArgument, TResult>(OperationVisitor<TArgument, TResult> visitor, TArgument argument)
+        {
+            return visitor.VisitAnonymousObjectCreationExpression(this, argument);
+        }
+    }
+
+    /// <summary>
+    /// Represents a C# or VB new/New anonymous object creation expression.
+    /// </summary>
+    internal sealed partial class AnonymousObjectCreationExpression : BaseAnonymousObjectCreationExpression, IAnonymousObjectCreationExpression
```

Comment:
Tagging @heejaechang again for more context, HeeJae can correct me if I am wrong - I think the idea here is to have a definition for both the Lazy and non-lazy versions of each Operation node, with the latter being useful for creating a fully populated tree, which is not being used currently, but we plan to use in future.